### PR TITLE
Changes for DDF and Alliance Installations

### PR DESCRIPTION
**Pull request by aaronilovici**
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by aaronilovici** [comment]
build now

**Review by cjlange — APPROVED** [review]

**Review by cjlange — APPROVED** [review]

**Comment by aaronilovici** [comment]
build now

File: README.md [hunk 1]
```diff
@@ -6,21 +6,17 @@ Replication is the process of creating a copy of a subset of data and storing it
...
-* Select the Install arrow for the replication feature.
-* The status will change from Uninstalled to Installed.
-
 #### Installing Via Kar Feature File
 An administrator can install the replication feature by following the following steps:
 * Locate and download the desired replication artifact kar file.
 * On a running and configured DDF instance, place the kar file in the ${DDF_HOME}/deploy directory.
 * Confirm the feature is running in the Admin Console or the Karaf command line interface.
 
+#### Additional Installation Steps
```
Comment (josephthweatt) [inline README.md]: Suggestion for title: **Additional Installation Steps for Alliance and DDF**

File: distributions/osgi/replication.policy [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+}
+
+grant codeBase "file:/ddf-adapter/replication-api-impl" {
+    permission java.lang.RuntimePermission "createClassLoader";
+}
+
+grant codeBase "file:/org.apache.servicemix.bundles.xstream" {
+    permission java.lang.RuntimePermission "createClassLoader";
+}
+
```
Comment (josephthweatt) [inline distributions/osgi/replication.policy:14]: I needed to add the following grant to get this working on DDF
```
grant codeBase "file:/replication-api-impl" {
    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}serverKeystore.jks", "read";
}
```
Would you mind adding it here?

Comment (aaronilovici) [reply distributions/osgi/replication.policy:14]: Roger, on it

**Comment by aaronilovici** [comment]
build now

**Review by josephthweatt — APPROVED** [review]

**Pull request merged**